metadata_backend/server.py

```diff
@@ -72,6 +72,7 @@ async def init() -> web.Application:
         web.get("/folders/{folderId}", rest_handler.get_folder),
         web.patch("/folders/{folderId}", rest_handler.patch_folder),
         web.delete("/folders/{folderId}", rest_handler.delete_folder),
+        web.get("/folders/publish/{folderId}", rest_handler.publish_folder),
```

Comment:
how about `/publish/{folderId}` ?

Reply:
and a `patch`

Reply:
also this might create an edge case with where `/folders/{folderId}` == /folders/publish`